src/lib/components/EventSchedule/index.js

```diff
@@ -177,12 +177,22 @@ class EventSchedule extends HTMLElement {
     this._tabsElement = null;
   }
 
-  onStateChanged({activeEventDay}) {
-    if (this._activeEventDay === activeEventDay) {
-      return;
-    }
+  onStateChanged({eventDays, activeEventDay}) {
     this._activeEventDay = activeEventDay;
 
+    eventDays.forEach(({isComplete, videoId}, index) => {
+      const el = this.querySelector(`[data-index="${index}"]`);
+      if (!el) {
+        return;
+      }
+      // TODO(samthor): Update description ("Coming soon" vs "All sessions")
+      if (isComplete) {
+        el.setAttribute('href', 'https://youtu.be/' + videoId);
```

Comment:
this is setting an href on the `.web-tabs__panel`, were you planning to use it for something?

Reply:
This whole section was actually my 1st attempt at this code. It didn't break so I didn't remove it. Gone now.